### PR TITLE
Fixes #35139 - package the foreman-maintain for Debian

### PR DESCRIPTION
**Pull request by upadhyeammit**

### CODE REVIEW

**Comment by upadhyeammit** [comment]
@evgeni maybe lots of things are incorrect, however here I start :)

File: debian/bullseye/foreman-maintain/files [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ruby-foreman-maintain_1.1.1-1_all.deb ruby optional
```
Comment (evgeni) [inline debian/bullseye/foreman-maintain/files:1]: please don't commit this file

File: debian/bullseye/foreman-maintain/ruby-foreman-maintain.substvars [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ruby:Versions=all
```
Comment (evgeni) [inline debian/bullseye/foreman-maintain/ruby-foreman-maintain.substvars:1]: please don't commit this file

File: debian/bullseye/foreman-maintain/ruby-foreman-maintain/DEBIAN/control [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Package: ruby-foreman-maintain
```
Comment (evgeni) [inline debian/bullseye/foreman-maintain/ruby-foreman-maintain/DEBIAN/control:1]: please do not commit any files in this folder

File: debian/bullseye/foreman-maintain/debhelper-build-stamp [hunk 4]
```diff
@@ -0,0 +1 @@
+ruby-foreman-maintain
```
Comment (evgeni) [inline debian/bullseye/foreman-maintain/debhelper-build-stamp:1]: drop that file too please (sorry, didn't notice it yesterday)

File: debian/bullseye/foreman-maintain/rules [hunk 5]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/make -f
+
+export GEM2DEB_TEST_RUNNER = --check-dependencies
```
Comment (evgeni) [inline debian/bullseye/foreman-maintain/rules:3]:
```suggestion
export GEM2DEB_TEST_RUNNER = --check-dependencies
export DH_RUBY = --gem-install
```

this should make the tests pass too

File: dependencies/bullseye/foreman_maintain/control [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+Section: ruby
+Priority: optional
+Maintainer: Amit Upadhye <upadhyeammit@gmail.com>
+Build-Depends: debhelper (>= 11~),
+               gem2deb,
+               ruby-clamp,
+               ruby-highline
+Standards-Version: 4.3.0
+Vcs-Git: git://github.com/theforeman/foreman_maintain.git
+Vcs-Browser: https://github.com/theforeman/foreman_maintain
```
Comment (evgeni) [inline dependencies/bullseye/foreman_maintain/control:11]:
```suggestion
```

these links don't carry debian packaging, and are also not useful in our case

File: dependencies/focal/foreman_maintain/control [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+Section: ruby
+Priority: optional
+Maintainer: Amit Upadhye <upadhyeammit@gmail.com>
+Build-Depends: debhelper (>= 11~),
+               gem2deb,
+               ruby-clamp,
+               ruby-highline
+Standards-Version: 4.3.0
+Vcs-Git: git://github.com/theforeman/foreman_maintain.git
+Vcs-Browser: https://github.com/theforeman/foreman_maintain
```
Comment (evgeni) [inline dependencies/focal/foreman_maintain/control:11]:
```suggestion
```

**Comment by evgeni** [comment]
can you add a `dirs` file with the following content please?
```
etc/foreman-maintain
var/lib/foreman-maintain
var/log/foreman-maintain
```

File: dependencies/bullseye/foreman_maintain/control [hunk 8]
```diff
@@ -0,0 +1,26 @@
+Source: ruby-foreman-maintain
+Section: ruby
+Priority: optional
+Maintainer: Amit Upadhye <upadhyeammit@gmail.com>
+Build-Depends: debhelper (>= 11~),
```
Comment (evgeni) [inline dependencies/bullseye/foreman_maintain/control:5]:
```suggestion
Build-Depends: debhelper (>= 12.8~),
```

File: dependencies/focal/foreman_maintain/control [hunk 9]
```diff
@@ -0,0 +1,26 @@
+Source: ruby-foreman-maintain
+Section: ruby
+Priority: optional
+Maintainer: Amit Upadhye <upadhyeammit@gmail.com>
+Build-Depends: debhelper (>= 11~),
```
Comment (evgeni) [inline dependencies/focal/foreman_maintain/control:5]:
```suggestion
Build-Depends: debhelper (>= 12.8~),
```

**Review by evgeni — APPROVED** [review]

**Comment by ehelms** [comment]
:fireworks:

**Pull request merged**